src/SapAct/Models/SQLTableDescriptor.cs

```diff
@@ -13,10 +13,16 @@ public SQLTableDescriptor GetChildTableDescriptor(string childTableName)
 	}
 
 	public bool IsEmpty => Columns.Count == 0 && ChildTables.Count == 0;
+
+	public SQLColumnDescriptor FindColumnCaseInsensitive(string columnName)
+	{
```

Comment:
should we check for null? or use the null conditional or coalescing operator here?

Reply:
I believe I have address these now - treating them as "public" methods

consuming code change to either throw exception of use null coalescing depending on where it makes logically sense